.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+# It uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+* @anselmbradford @cfarm @contollini @jimmynotjim @Scotchester
```

Comment:
It might be nice to include a comment making it clear that this is setting the default owners, and that more specific rules could be added below that would take precedence.

Reply:
Good point, I've reworked the doc a bit to better explain it and added a working rule for the deployment script.